pkgs/racket-test-core/tests/racket/portlib.rktl

```diff
@@ -14,14 +14,29 @@
   (test '(#\h #\e #\l #\l #\o #\return #\newline #\t #\h #\e #\r #\e)
 	port->list read-char (p))
   (test "hello\r\nthere" port->string (p))
+  (test "hello\r\nthere" port->string/close (p))
```

Comment:
I was using the wrong racket executable. Removed the offending lines and confirmed tests pass locally. Thanks!